kubemarine/resources/schemas/definitions/services/loadbalancer.json

```diff
@@ -94,6 +94,30 @@
           "description": "Target https port"
         }
       }
+    },
+    "keepalived": {
```

Comment:
Maybe add `"additionalProperties": false` everywhere to emphasize that we do not support other properties. Specifying of other properties makes sense for custom templates, but the properties can be hardcoded in the template in this case. Sections to consider:
* haproxy
* haproxy.global
* haproxy.defaults
* keepalived
* keepalived.global

Reply:
Added in every places in `loadbalancer` inventory schema